nextest-runner/src/target_runner.rs

```diff
@@ -273,3 +303,116 @@ impl TargetRunner {
         self.args.iter().map(AsRef::as_ref)
     }
 }
+
+fn deserialize_string_or_vec<'de, D>(deserializer: D) -> Result<Vec<String>, D::Error>
+where
+    D: serde::Deserializer<'de>,
+{
+    use serde::{
+        de::{self, value, SeqAccess, Visitor},
+        Deserialize,
+    };
+
+    struct StringOrVec;
+    impl<'de> Visitor<'de> for StringOrVec {
+        type Value = Vec<String>;
+
+        fn expecting(&self, formatter: &mut std::fmt::Formatter) -> std::fmt::Result {
+            formatter.write_str("string or list of strings")
+        }
+
+        fn visit_str<E>(self, s: &str) -> Result<Self::Value, E>
+        where
+            E: de::Error,
+        {
+            Ok(vec![s.to_owned()])
+        }
+
+        fn visit_seq<S>(self, seq: S) -> Result<Self::Value, S::Error>
+        where
+            S: SeqAccess<'de>,
+        {
+            Deserialize::deserialize(value::SeqAccessDeserializer::new(seq))
+        }
+    }
```

Comment:
Thanks! I think this should return an enum rather than a single string (which I believe cargo will interpret as a unit rather than splitting it up).

Reply:
Indeed, and it is way easier actually.